src/NzbDrone.Core.Test/Files/Indexers/HdBits/RecentFeed.json

```diff
@@ -28,7 +28,7 @@
             }
         },
         {
-            "id": 257140,
+            "id": "257140",
```

Comment:
We should test this in `HDBitsFixture` to make sure it works (it should, but let's make sure).

If we duplicate and rename this file to `RecentFeedStringIDs.json` and `RecentFeedLongIDs.json` and set the IDs appropriately we can both:

```
[Test]
        public void should_parse_recent_feed_from_HDBits()
        {
            var responseJson = ReadAllText(@"Files/Indexers/HdBits/RecentFeed.json");
```

becomes:

```
[TestCase("@"Files/Indexers/HdBits/RecentFeedLongIDs.json")]
[TestCase("@"Files/Indexers/HdBits/RecentFeedStringIDs.json")]
        public void should_parse_recent_feed_from_HDBits(string fileName)
        {
            var responseJson = ReadAllText(fileName);
```